packages/angular/src/schematics/add/index.ts

```diff
@@ -63,15 +116,46 @@ function addIonicStyles(projectName: string, projectSourceRoot: Path): Rule {
   };
 }
 
-function addIonicons(projectName: string): Rule {
+function addIonicons(projectName: string, projectSourceRoot: Path): Rule {
+  return (host: Tree) => {
+    const hasAppModule = host.exists(`${projectSourceRoot}/app/app.module.ts`);
+
+    if (hasAppModule) {
+      /**
+       * Add Ionicons to the `angular.json` file only if the project
+       * is using the lazy build of `@ionic/angular` with modules.
+       */
+      const ioniconsGlob = {
+        glob: '**/*.svg',
+        input: 'node_modules/ionicons/dist/ionicons/svg',
+        output: './svg',
+      };
+      addAsset(host, projectName, 'build', ioniconsGlob);
+      addAsset(host, projectName, 'test', ioniconsGlob);
+    }
+
+    return host;
+  };
+}
+
+function addIonicConfig(): Rule {
   return (host: Tree) => {
-    const ioniconsGlob = {
-      glob: '**/*.svg',
-      input: 'node_modules/ionicons/dist/ionicons/svg',
-      output: './svg',
-    };
-    addAsset(host, projectName, 'build', ioniconsGlob);
-    addAsset(host, projectName, 'test', ioniconsGlob);
+    const ionicConfig = 'ionic.config.json';
+    if (!host.exists(ionicConfig)) {
+      host.create(
+        ionicConfig,
+        JSON.stringify(
+          {
+            name: 'ionic-app',
+            app_id: '',
+            type: 'angular',
```

Comment:
For standalone apps the type should be `angular-standalone`: https://github.com/ionic-team/starters/blob/93bed6d18b652b7d81286a217351f74ee11bf0eb/angular-standalone/base/ionic.config.json#L4